supertokens_python/recipe/session/interfaces.py

```diff
@@ -362,7 +362,7 @@ async def verify_session(
         pass
 
 
-ResponseMutator = Callable[[BaseResponse], None]
+ResponseMutator = Callable[[BaseResponse], MaybeAwaitable[None]]
```

Comment:
Maybe just use async functions everywhere? I don't think there's any harm making all mutators async. But there's a big plus: linter will prbaobly catch any mistakes for now and for future (verify this once)

```suggestion
ResponseMutator = Callable[[BaseResponse], Awaitable[None]]
```